.docusaurus/docusaurus-plugin-content-docs/default/site-docs-install-index-md-c19.json

```diff
@@ -0,0 +1,28 @@
+{
+  "id": "install/index",
+  "title": "Getting Started",
+  "description": "Follow the steps to install Colony in a virtual Vagrant environment",
+  "source": "@site/docs/install/index.md",
+  "sourceDirName": "install",
+  "slug": "/install/",
+  "permalink": "/docs/next/install/",
+  "draft": false,
+  "unlisted": false,
+  "tags": [],
+  "version": "current",
+  "sidebarPosition": 2,
+  "frontMatter": {
+    "title": "Getting Started",
+    "description": "Follow the steps to install Colony in a virtual Vagrant environment",
+    "sidebar_position": 2
+  },
+  "sidebar": "myAutogeneratedSidebar",
+  "previous": {
+    "title": "Overview",
+    "permalink": "/docs/next/"
+  },
+  "next": {
+    "title": "Installing Colony",
+    "permalink": "/docs/next/install/virtual-install"
+  }
+}
```

Comment:
@konstructcarrie this "mark" here means that the code editor you're using isn't adding a standalone line at the end of each file as posix recommends.

If you use VSCode, this tutorial might help https://neilsonwebdesign.com/vs-code-force-newline-at-end-of-file/.

Reply:
VS Code has it listed in settings, thanks for the tip

Reply:
Updated, thanks.